File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.go
func twoSum(numbers []int, target int) []int {
    for i := 0; i< len(numbers); i++{
        for j := i+1; j <len(numbers); j++{
            if j != i{
                if numbers[j] + numbers[i] == target{
                    return []int{i+1,j+1}
                }
            }
        }
    }
    
    return []int{}
    
}

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, num: List[int], target: int) -> List[int]:
        
        a, b = 0, len(num) -1
        while a < b :
            curr = num[a] + num[b] 
            if curr == target:
                return [a+1, b+1]
            else:
                if curr < target:
                    a +=1
                else:
                    b -= 1
                
